flix-core/src/main/java/com/epam/mentoring/flixcore/controller/MovieController.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.mentoring.flixcore.controller;
+
+import com.epam.mentoring.flixcore.model.Movie;
+import com.epam.mentoring.flixcore.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.List;
+
+@Controller
+public class MovieController {
+
+    private static final String MOVIE_PATH_ROOT = "/movie/";
+
+    @Autowired
+    private MovieService movieService;
+
+    @RequestMapping(value = MOVIE_PATH_ROOT, method = RequestMethod.GET)
+    public ResponseEntity<List<Movie>> listAllMovies() {
+        List<Movie> movies = movieService.findAllMovies();
+        if(movies.isEmpty()){
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+        return new ResponseEntity<>(movies, HttpStatus.OK);
```

Comment:
Again this is fine, but there is builder provided that may be easily used.
For this sample something like:
return ResponseEntity.ok(movies).build();

Reply:
Done